Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -120,5 +120,16 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         XCTAssertNil(childViewController.parent)
     }
 
+    func testPresentPopover() {
+        let popover = UIViewController()
+        let presentingViewController = UIViewController()
+
+        presentingViewController.presentPopover(popover, sourcePoint: presentingViewController.view.center) {
+            XCTAssertEqual(presentingViewController.presentedViewController, popover)
```

Comment:
Are you sure that these even get called? The test may shut down before even getting here. You may need to use an `XCTestExpectation` to ensure that they're called.